includes/rewrites.php

```diff
@@ -58,15 +58,17 @@ function rewrite_entity_descr($descr)
 
 function ifNameDescr($interface, $device = null)
 {
-    return ifLabel($interface, $device);
+    return cleanPort($interface, $device);
 }
 
 
-function ifLabel($interface, $device = null)
+function cleanPort($interface, $device = null)
```

Comment:
can you add phpdoc here, mostly to signify that both variables should be arrays.